### DISCUSSION TITLE
DenseNet LayorNorm does not work!

### DISCUSSION THREAD

**Discussion by Hogandr** [Project-MONAI/MONAI#7978] [General]
In monai.networks.nets.DenseNet the layer norm does not work! 
monai.networks.nets.DenseNet121(spatial_dims=3, in_channels=1, out_channels=1, norm="LAYER")
=> TypeError: LayerNorm.__init__() missing 1 required positional argument: 'normalized_shape'

**Comment by KumoLiu** [comment]
Hi @Hogandr, please use tuple to add arguments.
Such as:
```
monai.networks.nets.DenseNet121(spatial_dims=3, in_channels=1, out_channels=1, norm=("layer", {"normalized_shape": [1]}))
```

Hope it helps, thanks.

**Reply by Hogandr** [reply]
@KumoLiu,
thank you very much! It worked for me :)
